packages/@react-aria/virtualizer/src/ScrollView.tsx

```diff
@@ -167,7 +169,7 @@ function ScrollView(props: ScrollViewProps, ref: RefObject<HTMLDivElement>) {
   }
 
   return (
-    <div {...otherProps} style={style} ref={ref} onScroll={onScroll}>
+    <div {...otherProps} style={style} ref={ref} onScroll={onScroll} data-rsp-testid="scrollview" >
```

Comment:
Test id that won't get overwritten by `data-testid` provided by user to ListView/other components that provide a `data-testid` to the grid. I've opted to not remove it via our babel config so others can mock `computedStyles` in the same way in their tests